go/vt/vtgate/planbuilder/operators/aggregation_pushing_helper.go

```diff
@@ -134,8 +134,8 @@ func (ab *aggBuilder) handleAggr(ctx *plancontext.PlanningContext, aggr Aggr) er
 		return ab.handlePushThroughAggregation(ctx, aggr)
 	case opcode.AggregateGroupConcat:
 		f := aggr.Func.(*sqlparser.GroupConcatExpr)
-		if f.Distinct || len(f.OrderBy) > 0 || f.Separator != "" {
-			panic("fail here")
```

Comment:
I understand now why it was crashing, the panic is not with vterrors.

Reply:
Yes, exactly!